research/size_of_problem/action_sequences.py

```diff
@@ -1,81 +1,58 @@
-from __future__ import annotations
+from typing import List, Tuple
```

Comment:
Cleaning up this file. I plan to use this in a test to validate my latest changes.